controllers/PostController.js

```diff
@@ -106,6 +106,26 @@ const unlikePost = async (req, res) => {
   }
 };
 
+const updatePost = async (req, res) => {
+  try {
+    const { postId, textContent, editedTimestamp } = req.body;
```

Comment:
Do we need the edited timestamp from the front end? We can make that on the backend in the controller and pass it into the query

Reply:
That makes sense, no need to pass information that can be generated from the backend. It will also be consistent with the way we do it in `addReply`

Reply:
Done